library/thrift

```diff
@@ -1,5 +1,7 @@
-# maintainer: Adam Hawkins <adam@hawkins.io> (@ahawkins)
+# maintainer: Adam Hawkins <hi@ahawkins.me> (@ahawkins)
 
 0.9: git://github.com/ahawkins/docker-thrift@d322572f7dd6ea468a14a4d832fbec26f152c71e 0.9
 0.9.3: git://github.com/ahawkins/docker-thrift@d322572f7dd6ea468a14a4d832fbec26f152c71e 0.9
-latest: git://github.com/ahawkins/docker-thrift@d322572f7dd6ea468a14a4d832fbec26f152c71e 0.9
+0.10: git://github.com/ahawkins/docker-thrift@e1f81dfe3e8fac5588e12d2b880166d1743dbecd 0.10.0
+0.10.0: git://github.com/ahawkins/docker-thrift@e1f81dfe3e8fac5588e12d2b880166d1743dbecd 0.10.0
+latest: git://github.com/ahawkins/docker-thrift@e1f81dfe3e8fac5588e12d2b880166d1743dbecd 0.9
```

Comment:
Also, CI is failing because you've specified a folder of `0.10.0` and the repo has just `0.10`. :+1:

(Is 0.9 actually still supported by upstream?)